README.md

```diff
@@ -202,12 +176,6 @@ You can deploy this project using:
 - GitHub: [@ijosue16](https://github.com/ijosue16)
 
 
-**👤 Author2**
-
-- GitHub: [@maccony202](https://github.com/maccony202)
-
-
-
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] OPTIONAL
MIT license link _on line 252_ in your ReadMe is broken, kindly endeavor to fix it, and remove the guide note below it.